nemo_skills/dataset/mmlu-redux/prepare.py

```diff
@@ -91,22 +91,28 @@ def format_entry(entry, category):
         # bad_question_clarity, bad_options_clarity, no_correct_answer,
         # multiple_correct_answers, expert and wrong_groundtruth with no labels
         return None
-    return {
-        "question": entry['question'],
+    new_entry = {
```

Comment:
if it's identical logic across a bunch of datasets, might make sense to move it to utils and import from there